src/schema/types/user.graphql

```diff
@@ -1,4 +1,21 @@
 type User {
   id: ID!
   name: String!
+  email: String!
+}
+
+input UserInput {
+  name: String!, 
+  email: String!, 
+  password: String!
+}
+
+extend type Query {
+  currentUser: User!
```

Comment:
currentUser may return undefined or null (no user logged in).

Reply:
Done